compiler/definitions/definition_path_for_annotation_repo.py

```diff
@@ -0,0 +1,9 @@
+from util import log
+
+PATH_ANNOTATION_REPO=None
```

Comment:
I would actually move this definition to `compiler/config.json`, which is the exact file to do such a configuration.

Reply:
I put it in `config.py`. Solving this is included in the `TODO.md`.

Reply:
Cool!